components/all/popups/ClassSchedulePopUpContent.tsx

```diff
@@ -2,25 +2,45 @@ import ShowMoreButton from "../buttons/ShowMoreButton";
 import ClassItemDetailed from "../ClassItemDetailed";
 import { useResponsive } from "../ResponsiveContext";
 
+interface Phase {
+  id: string;
+  name: string;
+}
+
+interface Module {
+  id: string;
+  name: string;
+  description: string;
+}
+
+interface ClassSchedulePopUpContainerProps {
+  phases: Phase[];
+  getModules: (phaseId: string) => Module[];
+  phaseTimes: Map<string, string>; // assuming phaseTimes is a map of phaseId to time string
+}
+
 export default function ClassSchedulePopUpContainer({
   phases,
   getModules,
   phaseTimes,
-}) {
-  const { isMobile, isTablet, isDesktop } = useResponsive();
+}: ClassSchedulePopUpContainerProps) {
+  const { isMobile } = useResponsive();
```

Comment:
i just kept all 3 as much as i could just in case we wanted to do that in the future